automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookBioconductorKernelSpec.scala

```diff
@@ -108,19 +108,19 @@ class NotebookBioconductorKernelSpec extends RuntimeFixtureSpec with NotebookTes
             notebookPage.executeCell("library('BiocSklearn')")
 
             notebookPage.executeCell("""BiocManager::install('rhdf5')""", installTimeout)
-            notebookPage.executeCell("library(rhdf5)")
+            notebookPage.executeCell("library('rhdf5')")
```

Comment:
@nturaga just curious, is `'` needed here?

Reply:
it shouldn't be https://stackoverflow.com/questions/25210455/how-can-library-accept-both-quoted-and-unquoted-strings